File: src/main/java/com/julien/juge/khast/api/dto/output/PostDto.java
package com.julien.juge.khast.api.dto.output;

import lombok.Builder;
import lombok.Data;

import java.util.Date;

@Data
@Builder
public class PostDto {

    private String id;

    private Date createdDate;

    private EventDto event;

    private TextDto text;

}

File: src/main/java/com/julien/juge/khast/api/dto/output/PostDto.kt
package com.julien.juge.khast.api.dto.output

import java.util.*

class PostDto {

    constructor(createdDate: Date?, id: String?, event: EventDto?, text: TextDto?)

    var id: String? = null
        set(id) {
            field = this.id
        }

    var createdDate: Date? = null
        set(createdDate) {
            field = this.createdDate
        }

    var event: EventDto? = null
        set(event) {
            field = this.event
        }

    var text: TextDto? = null
        set(text) {
            field = this.text
        }

}
